src/main/java/mat/HarpSupport.gwt.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module rename-to='harpsupport'>
+    <inherits name='com.google.gwt.user.User'/>
+    <set-property name="user.agent" value="ie9,ie10,safari,gecko1_8"/>
+    <inherits name='com.google.gwt.user.theme.standard.Standard'/>
+    <inherits name="org.gwtbootstrap3.GwtBootstrap3"/>
+    <inherits name="org.gwtbootstrap3.extras.respond.Respond"/>
+    <public path="resource">
+        <include name="css/*.*"/>
+        <include name="css/*.js"/>
+        <include name="fonts/*.*"/>
+        <include name="js/*.*"/>
+    </public>
+    <stylesheet src="css/mat-bootstrap-3.3.2.min.cache.css"/>
+    <!-- Other module inherits -->
+    <inherits name="com.google.gwt.logging.Logging"/>
+    <inherits name="com.google.gwt.xml.XML"/>
+    <inherits name="com.google.gwt.json.JSON"/>
+    <inherits name="org.gwtbootstrap3.extras.toggleswitch.ToggleSwitch"/>
+    <inherits name="org.gwtbootstrap3.extras.summernote.Summernote"/>
+    <inherits name='edu.ycp.cs.dh.acegwt.AceGWT'/>
```

Comment:
Few of these can be removed I guess from support module

Reply:
I tried to remove some of them and it failed. Looks like it's needed for java compilation to JS